src/material-experimental/mdc-slider/slider.spec.ts

```diff
@@ -1819,6 +1863,29 @@ class RangeSliderWithTwoWayBinding {
   endValue = 100;
 }
 
+@Component({
+  template: `
+  <div dir="rtl">
+    <mat-slider>
+      <input matSliderThumb>
+    </mat-slider>
+  </div>
+  `
+})
+class RTLSlider {}
+
+@Component({
+  template: `
+  <div dir="rtl">
+    <mat-slider>
+      <input matSliderStartThumb>
+      <input matSliderEndThumb>
+    </mat-slider>
+  </div>
+  `
+})
+class RTLRangeSlider {}
```

Comment:
You could avoid having to create different fixtures for RTL by providing the direction through DI. We have a few places where we do it this way already. Here's an example from `cdk/drag-drop`: https://github.com/angular/components/blob/master/src/cdk/drag-drop/directives/drag.spec.ts#L1892